packages/taquito-rpc/src/types.ts

```diff
@@ -1915,6 +1916,11 @@ export interface ConstantsResponseCommon {
 
 export type Ratio = { numerator: number; denominator: number };
 
+export interface ConstantsResponseProto017 extends ConstantsResponseProto016 {
+  attestation_threshold: number;
+  blocks_per_epoch: number;
```

Comment:
does it matter that these two added properties seem to belong under this parent property `dal_parametric`?

Reply:
good point, I'll adjust it instead so that 17 merely extends 16, and update the dal_parametric object to have the 2 new properties as optional properties.